stable-profiles/0.9.52/firejailed-tor-browser.profile

```diff
@@ -0,0 +1,117 @@
+# Firejail profile for firejailed-tor-browser
+# This file is overwritten after every install/update
+# Persistent local customizations
+include ${HOME}/.config/firejail/firejailed-tor-browser.local
+# Persistent global definitions
+include /etc/firejail/globals.local
+
+# Note: PluggableTransports didn't work with this profile
+
+noblacklist ${HOME}/.firejailed-tor-browser
+noblacklist ${HOME}/.config/firejail
+
+blacklist /opt
+blacklist /run/dbus/system_bus_socket
+blacklist /srv
+blacklist /usr/games
+blacklist /usr/local
+blacklist /usr/src
+
+include /etc/firejail/disable-common.inc
+include /etc/firejail/disable-devel.inc
+include /etc/firejail/disable-passwdmgr.inc
+include /etc/firejail/disable-programs.inc
+
+whitelist ${HOME}/.firejailed-tor-browser
+whitelist ${HOME}/.config/firejail
+whitelist /usr/share/alsa
+whitelist /usr/share/applications
+whitelist /usr/share/ca-certificates
+whitelist /usr/share/ca-certificates-java
+##whitelist /usr/share/crypto-policies
+##whitelist /usr/share/cursors
+whitelist /usr/share/dconf
+whitelist /usr/share/distro-info
+whitelist /usr/share/drirc.d
+whitelist /usr/share/enchant
+##whitelist /usr/share/enchant-2
+whitelist /usr/share/fontconfig
+whitelist /usr/share/fonts
+##whitelist /usr/share/gir-1.0
+##whitelist /usr/share/gjs-1.0
+whitelist /usr/share/glib-2.0
+whitelist /usr/share/glvnd
+##whitelist /usr/share/gtk-2.0
+##whitelist /usr/share/gtk-3.0
+whitelist /usr/share/gtksourceview-3.0
+##whitelist /usr/share/gtksourceview-4
+whitelist /usr/share/hunspell
+##whitelist /usr/share/hwdata
+whitelist /usr/share/icons
+whitelist /usr/share/knotifications5
+whitelist /usr/share/kservices5
+##whitelist /usr/share/Kvantum
+whitelist /usr/share/kxmlgui5
+whitelist /usr/share/libdrm
+whitelist /usr/share/libthai
+whitelist /usr/share/locale
+whitelist /usr/share/mime
+whitelist /usr/share/misc
+##whitelist /usr/share/Modules
+##whitelist /usr/share/myspell
+whitelist /usr/share/p11-kit
+whitelist /usr/share/pixmaps
+##whitelist /usr/share/pki
+##whitelist /usr/share/plasma
+##whitelist /usr/share/qt
+##whitelist /usr/share/qt4
+whitelist /usr/share/qt5
+whitelist /usr/share/sounds
+##whitelist /usr/share/tcl8.6
+whitelist /usr/share/tcltk/
+whitelist /usr/share/terminfo
+whitelist /usr/share/themes
+##whitelist /usr/share/thumbnail.so
+whitelist /usr/share/X11
+whitelist /usr/share/xml
+whitelist /usr/share/zoneinfo
+# Add the next two lines to firejailed-tor-browser.local to enable better desktop integration
+#include /etc/firejail/whitelist-common.inc
+#include /etc/firejail/whitelist-var-common.inc
```

Comment:
Either uncomment wvc or add `blacklist /var`.

Reply:
Ok. I will uncomment wvc here and then remove it from `firejailed-tor-browser.local`.